.github/workflows/integration_selenium.yaml

```
@@ -41,14 +53,21 @@ jobs:
         run: |
           echo "GALAXY_CONFIG_OVERRIDE_METADATA_STRATEGY=extended" >> $GITHUB_ENV
           echo "GALAXY_CONFIG_OVERRIDE_OUTPUTS_TO_WORKING_DIRECTORY=true" >> $GITHUB_ENV
-      - name: Prune unused docker image, volumes and containers
-        run: docker system prune -a -f
+      - if: github.event_name == 'workflow_dispatch'
+        run: |
+          echo "${{ github.event.inputs.env_vars }}" | tr " " "\n" >> $GITHUB_ENV
       - uses: actions/checkout@v2
+        if: github.event_name != 'workflow_dispatch'
         with:
           path: 'galaxy root'
-      - uses: actions/setup-python@v2
+      - uses: actions/checkout@v2
+        if: github.event_name == 'workflow_dispatch'
         with:
-          python-version: ${{ matrix.python-version }}
+          repository: ${{ github.event.inputs.repository }}
+          ref: ${{ github.event.inputs.ref }}
+          path: 'galaxy root'
```

This was deleted by mistake:

```suggestion
          path: 'galaxy root'
      - uses: actions/setup-python@v2
        with:
          python-version: ${{ matrix.python-version }}
```

Also, just noticed there's now a version 3 of https://github.com/actions/setup-python which has built-in caching of pip packages, so we can try that in a separate PR :)